haskell/cabal.bzl

```diff
@@ -610,6 +611,9 @@ haskell_cabal_binary = rule(
     _haskell_cabal_binary_impl,
     executable = True,
     attrs = {
+        "exe_name": attr.string(
+            doc = "Cabal executable component name. Defaults to name attribute.",
```

Comment:
That's incorrect, the value of `exe_name` defaults to the value of the `name` attribute. E.g. in `haskell_cabal_binary(name = "alex", ...)` it would default to `exe_name = "alex"`. I've spelled this out more explicitly.

Reply:
Urgh, yes. Reading too fast.